runtime/compiler/compile/J9SymbolReferenceTable.cpp

```diff
@@ -2563,3 +2564,64 @@ J9::SymbolReferenceTable::createParameterSymbol(
 
    return sym;
    }
+
+TR::SymbolReference *
+J9::SymbolReferenceTable::findArrayClassRomPtrSymbolRef()
+   {
+   return element(arrayClassRomPtrSymbol);
+   }
+
+TR::SymbolReference *
+J9::SymbolReferenceTable::findClassRomPtrSymbolRef()
+   {
+   return element(classRomPtrSymbol);
+   }
+
+const char *J9::SymbolReferenceTable::_commonNonHelperSymbolNames[] =
+   {
+   "<classRomPtr>",
+   "<ramStaticsFromClass>",
+   "<componentClassAsPrimitive>",
+   "<initializeStatusFromClass>",
+   "<threadPrivateFlags>",
+   "<arrayletSpineFirstElement>",
+   "<dltBlock>",
+   "<lowTenureAddress>",
+   "<highTenureAddress>",
+   "<fragmentParent>",
+   "<globalFragment>",
+   "<classFromJavaLangClassAsPrimitive>",
+   "<javaVM>",
+   "<heapBase>",
+   "<heapTop>",
+   "<j9methodExtraField>",
+   "<j9methodConstantPoolField>",
+   "<startPCLinkageInfo>",
+   "<instanceShapeFromROMClass>",
+   "<j9VMThreadTempSlotField>"
+   };
+
+
+const char *
+J9::SymbolReferenceTable::getNonHelperSymbolName(CommonNonhelperSymbol nonHelper)
+   {
+#ifdef OMR_ENABLE_NONHELPER_EXTENSIBLE_ENUM
```

Comment:
Note to reviewers: once this PR is committed, this macro will be defined upstream in OMR.  Once that macro definition propagates to OpenJ9 builds then this line can be removed.

Reply:
you meant to say "once https://github.com/eclipse/omr/pull/5822 is merged", right?

Reply:
uh, yes.   Not sure what I was thinking there.